extensions/positron-notebook-controllers/test-workspace/.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
An empty workspace is needed to update the config during tests.

Reply:
another curse of not having json comments.